lib/double_entry/reporting/aggregate.rb

```diff
@@ -25,7 +25,7 @@ def amount(force_recalculation = false)
         end
       end
 
-      def formatted_amount(value = amount())
+      def formatted_amount(value = amount)
```

Comment:
I think we could satisfy both world by doing the default value assignment in the body instead

```
def formatted_amount(value=nil)
  value ||= amount

```